torch/_inductor/cpp_builder.py

```diff
@@ -1545,3 +1547,65 @@ def build(self) -> Tuple[bytes, str]:
 
         _remove_dir(_build_tmp_dir)
         return status, self._target_file
+
+    def _remove_directory_from_file_path(self, file_path: str) -> str:
+        # Remove the directory part of file_path
+        return "${CMAKE_CURRENT_SOURCE_DIR}/" + Path(file_path).name
+
+    def save_compile_cmd_to_cmake(
+        self,
+        cmake_path: str,
+    ) -> None:
+        definitions = " ".join(self._build_option.get_definations())
+        contents = textwrap.dedent(
+            f"""
+            cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
+            project(aoti_model LANGUAGES CXX)
+
+            # May need to point CMAKE_PREFIX_PATH to the right torch location
+            find_package(Torch REQUIRED)
+
+            # Set a shared library target
+            add_library(aoti_model SHARED)
+
+            # Set the C++ standard
+            set(CMAKE_CXX_STANDARD 17)
+
+            # Set the compiler
+            set(CMAKE_CXX_COMPILER {self._compiler})
```

Comment:
Why this is is needed? Cmake should be able to auto-detect compiler you are using. This will also make AOTI C++ packaging non-portable, i.e. imagine it was generated on machine with gcc-9 as host compiler, but will be compiled on machine with gcc-10

Reply:
Let drop this.